labs/06_scale.md

```diff
@@ -98,13 +98,13 @@ Wie können Sie die Anzahl Replicas via Web Console steuern?
 Mit dem folgenden Befehl können Sie nun überprüfen, ob Ihr Service verfügbar ist, während Sie hoch- und herunterskalieren.
 Führen Sie folgenden Befehl in einem Terminal-Fenster aus und lassen ihn laufen, während Sie später skalieren.
 
-Ersetzen Sie `[HOSTNAME]` mit dem Hostname Ihrer definierten Route:
-
 __Tipp__:
+Um den entsprechenden Hostname anzuzeigen, kann folgender Befehl verwendet werden:
+
 `oc get route -o custom-columns=NAME:.metadata.name,HOSTNAME:.spec.host`
 
 ```bash
-while true; do sleep 1; ( { curl -fs http://[HOSTNAME]/health/; date "+ TIME: %H:%M:%S,%3N" ;} & ) 2>/dev/null; done
+export HNAME=$(oc get route -o custom-columns=:.spec.host | tail -n 1); while true; do sleep 1; ( { curl -fs http://${HNAME}/health/; date "+ TIME: %H:%M:%S,%3N" ;} & ) 2>/dev/null; done
```

Comment:
This command is too abstract in order for participants to easily understand what it does. Also, as soon as there's two routes inside the project (which already happened several times) it's not guaranteed that the correct route is taken.

Reply:
I'll revert it.